pallets/space-follows/src/benchmarking.rs

```diff
@@ -0,0 +1,72 @@
+//! Space follows pallet benchmarking.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+use frame_benchmarking::{account, benchmarks, whitelist, whitelisted_caller};
+use frame_support::{dispatch::DispatchError, ensure, traits::Currency};
+use frame_system::RawOrigin;
+use pallet_spaces::types::Space;
+use sp_std::vec;
+use subsocial_support::Content;
+
+fn create_dummy_space<T: Config>(
+    origin: RawOrigin<T::AccountId>,
+) -> Result<Space<T>, DispatchError> {
+    let space_id = pallet_spaces::NextSpaceId::<T>::get();
+
+    pallet_spaces::Pallet::<T>::create_space(origin.clone().into(), Content::None, None)
+        .map_err(|e| e.error)?;
+
+    let space = pallet_spaces::SpaceById::<T>::get(space_id)
+        .ok_or(DispatchError::Other("Space not found"))?;
+
+    Ok(space)
+}
+
+benchmarks! {
+
+    follow_space {
+        let space_owner_origin = RawOrigin::Signed(account::<T::AccountId>("SpaceOwner", 2, 0));
+        let space_follower = account::<T::AccountId>("SpaceFollower", 1, 0);
+
+        let space = create_dummy_space::<T>(space_owner_origin.clone())?;
+    }: {
+        Pallet::<T>::follow_space(
+            RawOrigin::Signed(space_follower.clone()).into(),
+            space.id,
+        )?;
+
+        // Cleanup
+        whitelist!(space_follower);
+        Pallet::<T>::unfollow_space(
+            RawOrigin::Signed(space_follower.clone()).into(),
+            space.id,
+        )?;
```

Comment:
Why do we need to unfollow a space in the benchmark for following space?

Reply:
I've tried to run benchmarks without it and got an error that I've already followed the space from the next iteration of the benchmarks.

And a fix was with this cleanup unfollow